apps/meteor/tests/end-to-end/api/22-push.ts

```diff
@@ -193,11 +193,21 @@ describe('[Push]', function () {
 	});
 
 	describe('[/push.info]', () => {
-		before(async () => {
-			await updateSetting('Push_enable', true);
-			await updateSetting('Push_enable_gateway', true);
-			await updateSetting('Push_gateway', 'https://random-gateway.rocket.chat');
-		});
+		before(async () =>
```

Comment:
does this have to be async? A Promise is already being returned here

Reply:
ouch, good catch. Thanks! 🤙